template/package.json

```diff
@@ -6,9 +6,9 @@
   "private": true,
   "scripts": {
     "dev": "node build/dev-server.js",
-    "start": "node build/dev-server.js",
+    "start": "npm run dev",
     "build": "node build/build.js"{{#unit}},
-    "unit": "cross-env BABEL_ENV=test karma start test/unit/karma.conf.js --single-run"{{/unit}}{{#e2e}},
+    "unit": "cross-env BABEL_ENV=test karma dev test/unit/karma.conf.js --single-run"{{/unit}}{{#e2e}},
```

Comment:
I'm not aware of a `karma dev` command. What is that?

Reply:
Nice code review. A mistake, fixing now :)